modules/shipit_scriptworker/files/requirements.txt

```diff
@@ -29,4 +29,4 @@ taskcluster==3.0.1
 urllib3==1.22
 virtualenv==16.0.0
 yarl==1.2.4
-shipitscript==1.0.0  # puppet: nodownload
+shipitscript==2.0.0  # puppet: nodownload
```

Comment:
FYI: I might have rolled-out some 2.0.0-dev work for this when testing with https://github.com/mozilla-releng/shipitscript/pull/8/files the other day. Sorry for that. I won't rollout that PR for another 2-3 weeks so feel free to rollout this and I'll bump again when the time comes.

Reply:
Shouldn't conflict. `-dev` will only be pulled in if asked for explicitly.

Reply:
Yeah, I agree. I mentioned it just in case you're wondering what a 2.0.0-devX package is doing there :)